nodejs/eks/cluster.ts

```diff
@@ -500,16 +500,6 @@ export class Cluster extends pulumi.ComponentResource {
      */
     public readonly instanceRole: pulumi.Output<aws.iam.Role>;
 
-    /**
-     * The security group for the cluster's nodes.
-     */
-    public readonly nodeSecurityGroup: aws.ec2.SecurityGroup;
-
-    /**
-     * The ingress rule that gives node group access to cluster API server
-     */
-    public readonly eksClusterIngressRule: aws.ec2.SecurityGroupRule;
```

Comment:
It looks like the `NodeGroupData` exposed as `defaultNodeGroup` does not include the ingress rule. Should it?  I can't think of any specific scenario for this - but its a resource we create that we used to make available but no longer will after these changes.

Reply:
Good point - I'll have to think on this one some more